src/components/molecules/SfBanner/SfBanner.spec.ts

```diff
@@ -0,0 +1,50 @@
+import { shallowMount } from "@vue/test-utils";
+import SfBanner from "@/components/molecules/SfBanner/SfBanner.vue";
+
```

Comment:
add scenario when we don't want button and uses call-to-action slot.
You should check:
- if default button is not displayed
- if call-to-action slot content is displayed